models/src/main/thrift/content/v1.thrift

```diff
@@ -395,6 +395,8 @@ struct PullquoteElementFields {
     2: optional string attribution
 
     3: optional string role
+
+    4: optional string source
```

Comment:
Would source fit better if it was on the EmbedTracking object rather than the Element?